apps/creds/templates/get_creds.html

```diff
@@ -0,0 +1,54 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block bannerTitle %}
+Blue Button 2.0 API Prod Access Credentialing
+{% endblock %}
+
+{% block bannerDescription %}
+The Blue Button 2.0 API Prod Access Credentialing allows you to obtain the OAUTH2.0 app credentials for your application.
+{% endblock %}
+
+{% block Content %}
+
+<div class="container">
+    <div class="ds-l-row ds-u-justify-content--center ds-u-margin-y--4">
+        <div class="ds-l-lg-col--11 ds-l-md-col--11 ds-l-sm-col--12">
+
+            <h2>User: {{ user_name }}</h2>
```

Comment:
Lets leave this off, the user_name is auto generated in prod, so it wont look familiar to them at all

Reply:
prefer to show user-org-app trio to reflect the User Org Apps conceptual models, and also pave way for re-credentialing existing apps...just my 2c

Reply:
This page will be seen by production app users though. Since we auto create the username with an `@example` email address it is only going to be confusing for them if they see it. I would agree that it could be useful in an environment where they sign up with their own email (like sandbox) but since this will only be used in prod, it'll only be confusing.

Reply:
Agree that the username should not show here in the final version of the template.